email_confirm_la/views.py

```diff
@@ -21,4 +22,8 @@ def confirm_email(request, confirmation_key):
     except EmailConfirmation.ExpiredError:
         return render(request, 'email_confirm_la/email_confirmation_expiration.html', context)
 
+    if configs.EMAIL_CONFIRM_LA_AUTOLOGIN:
+        email_confirmation.content_object.backend = 'django.contrib.auth.backends.ModelBackend'
+        login(request, email_confirmation.content_object)
```

Comment:
You should check whether if `email_confirmation.content_object`is a `User` object, after all `content_object` is a `GenericForeignKey`.